src/main/webapp/i18n/en/global.json

```
@@ -414,20 +410,36 @@
           "detail": "Cannot submit application: User authentication required."
         },
         "saveFailed": {
-          "summary": "Error",
-          "detail": "Failed to save application."
+          "summary": "Save failed",
+          "detail": "Changes could not be saved. Please check your connection and save again."
```

nitpick: since user's don't have to press a button to save the changes, it could be misleading to say "save" again
```suggestion
          "detail": "Changes could not be saved. Please check your connection and try again."
```

---

thanks 👍